tests/tas-elasticsearch/src/test/java/org/alfresco/elasticsearch/SearchQueryService.java

```diff
@@ -44,6 +45,12 @@ public void expectSomeResultsFromQuery(SearchRequest searchRequest, UserModel te
         expectResultsFromQuery(searchRequest, testUser, assertNotEmpty);
     }
 
+    public void expectResultsInOrder(SearchRequest searchRequest, UserModel user, List<String> expected)
```

Comment:
I think it will be easier to call the method if we use `String... expected` as in the other tests below.